src/display/components/total_bandwidth.rs

```diff
@@ -8,18 +8,30 @@ use crate::display::{DisplayBandwidth, UIState};
 
 pub struct TotalBandwidth<'a> {
     pub state: &'a UIState,
+    pub paused: bool,
 }
 
 impl<'a> TotalBandwidth<'a> {
     pub fn render(&self, frame: &mut Frame<impl Backend>, rect: Rect) {
-        let title_text = [Text::styled(
-            format!(
-                " Total Rate Up / Down: {} / {}",
-                DisplayBandwidth(self.state.total_bytes_uploaded as f64),
-                DisplayBandwidth(self.state.total_bytes_downloaded as f64)
-            ),
-            Style::default().fg(Color::Green).modifier(Modifier::BOLD),
-        )];
+        let title_text = [if self.paused {
+            Text::styled(
+                format!(
+                    " Total Rate Up / Down: {} / {} [PAUSED]",
+                    DisplayBandwidth(self.state.total_bytes_uploaded as f64),
+                    DisplayBandwidth(self.state.total_bytes_downloaded as f64)
+                ),
+                Style::default().fg(Color::Yellow).modifier(Modifier::BOLD),
+            )
+        } else {
+            Text::styled(
+                format!(
+                    " Total Rate Up / Down: {} / {}",
```

Comment:
This seems rather verbose. Shouldn't we optionally append the `[PAUSED]` string if `self.paused`?

Reply:
Unfortunately rust requires that format strings must be literals... Plus, the color argument will have to be different, so I just expanded this expression